content/pages/roadmap.html

```diff
@@ -16,25 +16,186 @@ <h4><i class="fa fa-map"></i> <b>Roadmap</b></h2>
+                                <li>Every server becomes a Provider</li>
+                            </ul>
+                        </li>
+
+                        <li>
+                            Improve support for archive files
+                            <ul>
+                                <li>
+                                    Make archive entries visible in the catalog
+                                    <ul>
+                                        <li>Design new abstractions, semantics, and APIs</li>
+                                    </ul>
+                                </li>
+                                <li>Deprecate all functionality stemming from special collections and imcoll</li>
+                            </ul>
+                        </li>
+                    </ul>
+
+                    <h3>Plugins and Capabilities</h3>
+
+                    <ul>
+                        <li>Finish and release the Policy Composition rule engine plugin</li>
+                        <li>Finish and release the Publishing capability</li>
+                    </ul>
+
+                    <h3>Libraries</h3>
+
+                    <ul>
+                        <li>
+                            Add support for new Authentication Plugin Framework introduced in the iRODS 4.3 series
+                            <ul>
+                                <li>All client-side libraries, regardless of the programming language, must be updated</li>
+                                <li>
+                                    Required for <strong>pam_interactive</strong> authentication scheme
+                                    <ul>
+                                        <li>This becomes the default for iCommands</li>
+                                    </ul>
+                                </li>
+                            </ul>
+                        </li>
+                    </ul>
+
+                    <h3>Clients</h3>
+
+                    <ul>
+                        <li>
+                            Replace iCommands with a new CLI
+                            <ul>
+                                <li>A single binary backed by a plugin architecture allowing its functionality to be extended</li>
+                                <li>No memory leaks and properly handles signals</li>
+                                <li>Help text and documentation will be consistent in formatting and detail</li>
+                            </ul>
+                        </li>
+
+                        <li>
+                            Improve Metalnx
+                            <ul>
+                                <li>Add support for the <strong>pam_interactive</strong> authentication scheme</li>
+                                <li>Remove the need for helper database</li>
+                                <li>
+                                    Remove all administrative features
```

Comment:
"all" -> "most"

Reply:
Most? What features would possibly remain?

Reply:
change ownership/permissions on data objects?   queries that know you're an admin... -M type stuff...

Reply:
Done